src/resources/postcss/customizer-controls.pcss

```
@@ -0,0 +1,304 @@
+
+			&::-ms-track {
+				background-color:  #FFF;
+				border-radius: 9px;
+				border: 0;
+				cursor: pointer;
+				height: 18px;
+				width: 100%;
+			}
+
+			&::-moz-range-progress {
+				background-color:  #334AFF;
+				border-radius: 9px;
+				height: 16px;
+			}
+
+			&::-webkit-progress-value {
+				background-color:  #334AFF;
+			}
+
+			&::-ms-fill-lower {
+				background-color:  #334AFF;
+			}
+
+			/* Thumb styles for allthebrowsers */
+
+			&::-moz-range-thumb {
+				background: #ffffff;
+				border-radius: 9px;
+				border: 1px solid #141827;
+				cursor: pointer;
+				height: 16px;
+				width: 16px;
+
+				&:active,
+				&:focus {
+					border-radius: 12px;
+					height: 24px;
+					width: 24px;
+				}
+			}
+
+			&::-webkit-slider-thumb {
+				background: #ffffff;
+				border-radius: 9px;
+				border: 1px solid #141827;
+				cursor: pointer;
+				height: 16px;
+				width: 16px;
+
+				&:active,
+				&:focus {
+					border-radius: 12px;
+					height: 24px;
+					width: 24px;
+				}
+			}
+
+			&::-ms-thumb {
+				background: #ffffff;
+				border-radius: 9px;
+				border: 1px solid #141827;
+				cursor: pointer;
+				height: 16px;
+				width: 16px;
+
+				&:active,
+				&:focus {
+					border-radius: 12px;
+					height: 24px;
+					width: 24px;
+				}
+			}
+		}
+
+		.tec-range-slider-datalist {
+			display: flex;
+			justify-content: space-between;
+			width: calc(100% - 24px);
+		}
+
+		.tec-range-slider-option {
+			text-align: center;
+
+			&:first-child {
+				text-align: left;
+			}
+
+			&:last-child {
+				text-align: right;
+			}
+		}
+
+		/* Toggle Switch, TEC-style */
+
+		&.customize-control-toggle {
+
+			.customize-inside-control-row{
+				margin-left: 0;
+				padding-left: 0;
+				display: flex;
+				justify-content: space-between;
+			}
+
+			.tec-switch-label {
+				display: inline-flex;
+				align-items: center;
+				margin: 5px 0;
+
+				&:active,
+				&:focus,
+				&:focus-visible {
+					outline: none;
+
+					.tec-switch-toggle {
+						box-shadow: 0 0 0 2px #334AFF;
+					}
+				}
+			}
+
+			.tec-switch-toggle {
+				border-radius: 12px;
+				cursor: pointer;
+				font-size: 16px;
+				position: relative;
+				padding: 2px;
+
+				&:before,
+				&:after {
+					content: '';
+					margin: 0;
+					transition: all 100ms cubic-bezier(0.4, 0.0, 0.2, 1);
+					display: block;
+				}
+
+				&:before {
+					background-color: #F7F6F6;
+					border: 1px solid #BABABA;
+					height: 1.125em;
+					width: 2.75m;
+					border-radius: .55em;
+					opacity: 0.6;
+				}
+
+				&:after {
+					background-color: #D5D5D5;
+					position: absolute;
+					top: 50%;
+					transform: translate(0, -50%);
+					height: 16px;
+					width: 16px;
+					border-radius: 50%;
+					left: 3px;
+				}
+			}
+
+			.tec-switch-input {
+				border: 0;
+				clip: rect(0 0 0 0);
+				height: 1px;
+				margin: -1px;
+				overflow: hidden;
+				padding: 0;
+				position: absolute;
+				width: 1px;
+
+				&:checked {
+					+.tec-switch-toggle {
```

```suggestion
					+ .tec-switch-toggle {
```
this and all the other ones below